emojis.json

```diff
@@ -2819,11 +2819,306 @@
+    "char": "🇧🇯",
+    "category": "flags"
+  },
+  "bm": {
+    "keywords": ["bermuda", "flag", "nation", "country", "banner"],
+    "char": "🇧🇲",
+    "category": "flags"
+  },
+  "bt": {
+    "keywords": ["bhutan", "flag", "nation", "country", "banner"],
+    "char": "🇧🇹",
+    "category": "flags"
+  },
+  "bo": {
+    "keywords": ["bolivia", "flag", "nation", "country", "banner"],
+    "char": "🇧🇴",
+    "category": "flags"
+  },
+  "bq": {
+    "keywords": ["bonaire", "flag", "nation", "country", "banner"],
+    "char": "🇧🇶",
+    "category": "flags"
+  },
+  "ba": {
+    "keywords": ["bosnia", "herzegovina", "flag", "nation", "country", "banner"],
+    "char": "🇧🇦",
+    "category": "flags"
+  },
+  "bw": {
+    "keywords": ["botswana", "flag", "nation", "country", "banner"],
+    "char": "🇧🇼",
+    "category": "flags"
+  },
+  "br": {
+    "keywords": ["brazil", "flag", "nation", "country", "banner"],
+    "char": "🇧🇷",
+    "category": "flags"
+  },
+  "io": {
+    "keywords": ["british", "indian", "ocean", "territory", "flag", "nation", "country", "banner"],
+    "char": "🇮🇴",
+    "category": "flags"
+  },
+  "vg": {
+    "keywords": ["british", "virgin", "islands", "bvi", "flag", "nation", "country", "banner"],
+    "char": "🇻🇬",
+    "category": "flags"
+  },
+  "bn": {
+    "keywords": ["brunei", "darussalam", "flag", "nation", "country", "banner"],
+    "char": "🇧🇳",
+    "category": "flags"
+  },
+  "bg": {
+    "keywords": ["bulgaria", "flag", "nation", "country", "banner"],
+    "char": "🇧🇬",
+    "category": "flags"
+  },
+  "bf": {
+    "keywords": ["burkina", "faso", "flag", "nation", "country", "banner"],
+    "char": "🇧🇫",
+    "category": "flags"
+  },
+  "bi": {
+    "keywords": ["burundi", "flag", "nation", "country", "banner"],
+    "char": "🇧🇮",
+    "category": "flags"
+  },
+  "cv": {
+    "keywords": ["cabo", "verde", "flag", "nation", "country", "banner"],
+    "char": "🇨🇻",
+    "category": "flags"
+  },
+  "kh": {
+    "keywords": ["cambodia", "flag", "nation", "country", "banner"],
+    "char": "🇰🇭",
+    "category": "flags"
+  },
+  "cm": {
+    "keywords": ["cameroon", "flag", "nation", "country", "banner"],
+    "char": "🇨🇲",
+    "category": "flags"
+  },
+  "ca": {
+    "keywords": ["canada", "flag", "nation", "country", "banner"],
+    "char": "🇨🇦",
+    "category": "flags"
+  },
+  "ic": {
+    "keywords": ["canary", "islands", "flag", "nation", "country", "banner"],
+    "char": "🇮🇨",
+    "category": "flags"
+  },
+  "ky": {
+    "keywords": ["cayman", "islands", "flag", "nation", "country", "banner"],
+    "char": "🇰🇾",
+    "category": "flags"
+  },
+  "cf": {
+    "keywords": ["central", "african", "republic", "flag", "nation", "country", "banner"],
+    "char": "🇨🇫",
+    "category": "flags"
+  },
+  "td": {
+    "keywords": ["chad", "flag", "nation", "country", "banner"],
+    "char": "🇹🇩",
+    "category": "flags"
+  },
+  "chile": {
```

Comment:
Most of the country flags use the ISO 3166-2 country code as the emoji code. Chile's ISO code is "cl", but ["cl" is already used by one of the previously-existing emojis](https://github.com/jasonrudolph/emojilib/blob/b0ce5ed37f3fefb42c4800f4398f430f9cc9392f/emojis.json#L3942), so I'm proposing that we use "chile" as the emoji code here.